lib/Workflow/Persister/DBI/AutoGeneratedId.pm

```diff
@@ -13,7 +13,6 @@ __PACKAGE__->mk_accessors(@FIELDS);
 
 sub new {
     my ( $class, $params ) = @_;
-    my $log = get_logger();
```

Comment:
With the removal of this statement, I cannot see where the log handle comes from in this class, it does not inherit from any classes provided the log handle via the main object - or am I missing something?

Reply:
You're totally right. Adding the `log` attribute to both classes and further aligning the two (one has a "new" method, the other one doesn't...)

Reply:
I've replaced this now with a `log` accessor in the class which is initialized upon instantiation.